src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: *sync* ot *hybrid* mode. The command affected by this is the [`okteto up`](/reference/okteto-cli.mdx/#up) command being configurable at manifest level.
+
+The default mode used is the *sync* mode in which changes made to the local service are synchronized with the remote development container deployed in the cluster. On the other hand, the *hybrid* mode allows to run the development service locally and the rest of the application components in the cloud.
+
+### Problem to solve
+
+Sometimes running the service locally is faster than the original approach taken in okteto where the service runs in the cloud and changes are synchronized from local to the remote development container. Another possibility can be if you’re using a framework that is IDE-heavy like on Java/Spring, IDEs tend to be pre-configured to run tests, start services, and connect debuggers. You lose some of that ‘click and you are done’ when you  run your development service in the cluster. This is where hybrid development comes in.
```

Comment:
```suggestion
There are situations when running a service locally is faster than synchronizing code with a remote development container. For example: when doing frontend-only development or when working in an IDE-heavy context (e.g. Java/Spring) where the IDE may be pre-configured to run tests, start services, and connect debuggers. 

When using Okteto's `hybrid` mode you can maintain your local development experience for the specific service you're working on while still leveraging the benefits of hosting the rest of your application's components in the remote development container.
```

Wondering if something like this might work?

Reply:
af9a32b317861e294945db0b4e0dd006fdc2c9e8